src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelperOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.TagHelpers
+{
+    /// <summary>
+    /// Options pertaining to the default behavior of <see cref="FormTagHelper"/> instances.
+    /// </summary>
+    public class FormTagHelperOptions
+    {
+        /// <summary>
+        /// Whether the anti-forgery token should be generated by default for all instances of
+        /// <see cref="FormTagHelper"/>. Can be overridden on any given instance.
+        /// </summary>
+        /// <value>
+        ///     Defaults to <c>null</c>, which indicates a token will only be generated if the <c>action</c>
+        ///     attribute was not explicitly defined.
+        /// </value>
+        public bool? AntiForgery { get; set; }
```

Comment:
The property name isn't terribly descriptive. Imagine seeing code that said `options.AntiForgery = true` - what would that mean? Should this name be `IncludeAntiForgeryToken`? (Kinda verbose, but more descriptive.)